src/cursor/cur_backup_incr.c

```diff
@@ -144,14 +148,21 @@ __curbackup_incr_next(WT_CURSOR *cursor)
              */
             WT_ERR(__curbackup_incr_blkmod(session, btree, cb));
             /*
-             * If there is no block modification information for this file, this is a newly created
-             * file without any checkpoint information. Return the whole file information.
+             * If there is no block modification information for this file, it's either a newly
+             * created file without any checkpoint information, or a file created and checkpointed
+             * before backups were configured and not subsequently modified. In the first case, we
+             * return the whole file information. We ignore the file in the second, as files created
+             * before configuring backups are copied as part of the initial full backup.
              */
             if (cb->bitstring.mem == NULL) {
-                WT_ERR(__wt_fs_size(session, cb->incr_file, &size));
                 cb->incr_init = true;
-                __wt_cursor_set_key(cursor, 0, size, WT_BACKUP_FILE);
-                goto done;
+                if (cb->has_ckpt)
+                    WT_ERR(WT_NOTFOUND);
+                else {
```

Comment:
I don't think there's any need for the `else`.

Reply:
Agreed & fixed.